docs/pages/database-access/guides/oracle-self-hosted.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Database Access with Oracle (Preview)
+description: How to configure Teleport Database Access with Oracle.
+---
+
+<Admonition type="warning" title="Preview">
+  Database Access for Oracle Server is currently in Preview mode.
```

Comment:
I'm not really familiar with Oracle—is Oracle Server a separate product than Oracle? Is database access for Oracle generally available but for Oracle Server it's in preview?

Reply:
Yes that is misleading  It should refer to  Oracle database and there is not Oracle product called "Oracle Server". 
Updated.